pkg/edit/complete/util_unix.go

```diff
@@ -0,0 +1,5 @@
+// +build !windows,!plan9
+
+package complete
+
+const IsCaseInsensitiveOs bool = false
```

Comment:
Ugh! MacOS (a UNIX family OS and which I use on my home server and laptop) is case preserving by default; a special-case of case insensitive. So this is wrong on my systems. It's also why I suggested that this sort of change should be discussed before being implemented. I don't understand why this behavior depends on the platform. Especially since, as you noted in your commit comment, that some platforms may allow mounting a filesystem with case insensitive behavior even if the underlying filesystem is case sensitive.